src/server/scripts/Northrend/UtgardeKeep/UtgardePinnacle/boss_svala.cpp

```diff
@@ -377,11 +376,13 @@ class boss_svala : public CreatureScript
                             break;
                         case EVENT_FINISH_RITUAL:
                             me->SetDisableGravity(false, true);
-                            SetCombatMovement(true);
-
+                            me->GetMotionMaster()->MoveFall();
```

Comment:
`me->SetDisableGravity(false, true);` probably should already make unit fall

Reply:
thx